arch/arc/Kconfig

```diff
@@ -126,8 +126,7 @@ config	FAULT_DUMP
 	  0: Off.
 
 config	XIP
```

Comment:
BTW, don't we need a "bool" here?

Reply:
Only need to give the type once for symbols defined in multiple locations. Doesn't hurt to repeat it though.

Looks like the "base" definition of `XIP` is in `kernel/Kconfig`: http://docs.zephyrproject.org/reference/kconfig/CONFIG_XIP.html

Reply:
Alright, thanks for the clarification. For ease of reading , I would add the type here, too.